webcomponents/tool/src/main/frontend/packages/sakai-notifications/src/SakaiNotifications.js

```diff
@@ -197,39 +232,44 @@ export class SakaiNotifications extends SakaiElement {
           this._fireLoadedEvent();
           this._filterIntoToolNotifications(false);
         } else {
-          console.error(`Failed to clear notification with id ${notificationId}`);
+          throw new Error(`Network error while clearing notification at ${url}`);
         }
-      });
+      })
+      .catch(error => console.error(error));
   }
 
   _clearAllNotifications() {
 
-    fetch("/direct/portal/clearAllNotifications", { cache: "no-store", credentials: "include" })
+    const url = "/api/users/me/notifications/clear";
+    fetch(url, { method: "POST", cache: "no-store", credentials: "include" })
       .then(r => {
 
         if (r.ok) {
           this.notifications = [];
           this._fireLoadedEvent();
           this._filterIntoToolNotifications();
         } else {
-          console.error("Failed to clear all notifications");
+          throw new Error(`Network error while clearing all notifications at ${url}`);
         }
-      });
+      })
+      .catch(error => console.error(error));
   }
 
   _markAllNotificationsViewed() {
 
-    fetch("/direct/portal/markAllNotificationsViewed", { cache: "no-store", credentials: "include" })
+    const url = "/api/users/me/notifications/markViewed";
+    fetch(url, { method: "POST", cache: "no-store", credentials: "include" })
```

Comment:
cache header prob not necessary as this is a POST

Reply:
Good spot, thanks.